packages/tldraw/src/lib/shapes/embed/EmbedShapeUtil.tsx

```diff
@@ -127,6 +129,25 @@ export class EmbedShapeUtil extends BaseBoxShapeUtil<TLEmbedShape> {
 
 		const pageRotation = this.editor.getShapePageTransform(shape)!.rotation()
 
+		if (svgExport) {
+			// for SVG exports, we show a blank embed
+			return (
+				<HTMLContainer className="tl-embed-container" id={shape.id}>
```

Comment:
Maybe? I think it's tricky because we do sort of allocate space for it in the document. Right now we just render a blank rectangle in exports. Mind if i leave this as-is now as this is the same as the existing behaviour? We can follow up if we decide we want to do something like this

Reply:
cool and yes! it's totally fine for now, something to think about for future PRs